src/VisualStudio/Core/Test/Diagnostics/DefaultDiagnosticUpdateSourceTests.vb

```diff
@@ -29,7 +29,7 @@ class 123 { }
                 Dim listenerProvider = workspace.ExportProvider.GetExportedValue(Of IAsynchronousOperationListenerProvider)
                 Dim diagnosticService = New DiagnosticService(listenerProvider)
 
-                Dim miscService = New DefaultDiagnosticAnalyzerService(diagnosticService)
+                Dim miscService = New DefaultDiagnosticAnalyzerService(New TestDiagnosticAnalyzerService(DiagnosticExtensions.GetCompilerDiagnosticAnalyzersMap()), diagnosticService)
```

Comment:
📝 `DefaultDiagnosticAnalyzerService` is a MEF-exported type. It should be obtained from `workspace.ExportProvider` and not directly constructed. Non-blocking since this PR did not introduce the incorrect usage.